src/main/java/core/basesyntax/service/FileReader.java

```diff
@@ -0,0 +1,4 @@
+package core.basesyntax.service;
+
+public abstract class FileReader implements Readable {

```

Comment:
Any code here, why do we need this redundant layer of abstraction?
```suggestion
public abstract class FileReader implements Readable {

```